clang/lib/Driver/ToolChains/Assigner.h

```diff
@@ -50,12 +50,12 @@ class LLVM_LIBRARY_VISIBILITY Assigner final : public ToolChain {
                           options::OPT_nostdlibinc)) {
       return;
     }
-    addExternCSystemInclude(DriverArgs, CC1Args, "/usr/include/zkllvm");
+    addExternCSystemInclude(DriverArgs, CC1Args, "/usr/lib/zkllvm/include/libc");
```

Comment:
I am concerned about hardcoding the paths. Is there maybe any other ways to do that?

Reply:
I discussed it with @makxenov. We decided to keep it like that for now. Looks like the LLVM project uses the same approach but uses two hardcoded paths  - one for the gcc and one for the clang.